ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/CipherSuitesJvm.kt

```diff
@@ -8,5 +8,7 @@ import io.ktor.network.tls.platform.*
 
 internal actual fun CipherSuite.isSupported(): Boolean = when (platformVersion.major) {
     "1.8.0" -> platformVersion.minor >= 161 || keyStrength <= 128
-    else -> keyStrength <= 128
+    "1.7.0" -> platformVersion.minor >= 171 || keyStrength <= 128
+    "1.6.0" -> platformVersion.minor >= 181 || keyStrength <= 128
+    else -> true
```

Comment:
How does this looks @e5l 

Used this https://www.oracle.com/java/technologies/javase/8u161-relnotes.html as it gives security baseline. 

I still am not able to find out what key strength is supported for these earlier versions, but from jdk 9 onwards we should be able to support any key strength right ?

Reply:
Yep, it looks good